elementary/monitor/fetchers/alerts/normalized_alert.py

```diff
@@ -90,31 +108,23 @@ def _get_alert_meta_attrs(self, meta_key: str) -> List[Optional[str]]:
             attrs.append(model_attrs)
         return attrs
 
-    def _get_alert_chennel(self) -> Optional[str]:
-        model_slack_channel = self.model_meta.get(CHANNEL_KEY)
-        test_slack_channel = self.test_meta.get(CHANNEL_KEY)
-        return test_slack_channel or model_slack_channel
+    def _get_alert_channel(self) -> Optional[str]:
+        return self._get_field_from_test_meta_or_model_meta_or_default(key=CHANNEL_KEY)
 
     def _get_alert_suppression_interval(self) -> int:
-        model_alert_suppression_interval = self.model_meta.get(
-            ALERT_SUPRESSION_INTERVAL_KEY
-        )
-        test_alert_suppression_interval = self.test_meta.get(
-            ALERT_SUPRESSION_INTERVAL_KEY
+        return self._get_field_from_test_meta_or_model_meta_or_default(
+            key=ALERT_SUPRESSION_INTERVAL_KEY, default_val=0
         )
-        if test_alert_suppression_interval is not None:
-            return test_alert_suppression_interval
-        elif model_alert_suppression_interval is not None:
-            return model_alert_suppression_interval
-        else:
-            return 0
 
     def _get_alert_fields(self) -> Optional[List[str]]:
         # If there is no alerts_fields in the test meta object,
         # we return the model alerts_fields from the model meta object.
         # The fallback is DEFAULT_ALERT_FIELDS.
-        return (
-            self.test_meta.get(ALERT_FIELDS_KEY)
-            or self.model_meta.get(ALERT_FIELDS_KEY)
-            or DEFAULT_ALERT_FIELDS
+        return self._get_field_from_test_meta_or_model_meta_or_default(
+            key=ALERT_FIELDS_KEY, default_val=DEFAULT_ALERT_FIELDS
         )
+
+    def _get_field_from_test_meta_or_model_meta_or_default(
```

Comment:
Please add return typing (could be even Any)

Reply:
Agreed

Reply:
OK

Reply:
done